empiric-package/empiric/core/client.py

```diff
@@ -63,7 +78,7 @@ async def get_value(self, key, aggregation_mode, sources=None):
             response.num_sources_aggregated,
         )
 
-    async def get_entries(self, key, sources=None):
+    async def get_entries(self, key, sources=None) -> Any:
```

Comment:
same here

Reply:
List of `Entry`s (I think), where `Entry` is the namedtuple defined in types